sql.go

```diff
@@ -145,17 +145,17 @@ func (s *sql) UpdatePrefix(prefix Prefix) (Prefix, error) {
 	if rows == 0 {
 		err := tx.Rollback()
 		if err != nil {
-			return Prefix{}, newOptimisticLockError("updatePrefix did not effect any row, but rollback did not work:" + err.Error())
+			return Prefix{}, fmt.Errorf("updatePrefix did not effect any row, but rollback did not work:%w", err)
 		}
-		return Prefix{}, newOptimisticLockError("updatePrefix did not effect any row")
+		return Prefix{}, fmt.Errorf("%w: updatePrefix did not effect any row", ErrOptimisticLockError)
 	}
 	return prefix, tx.Commit()
 }
 
 func (s *sql) DeletePrefix(prefix Prefix) (Prefix, error) {
 	tx, err := s.db.Beginx()
 	if err != nil {
-		return Prefix{}, fmt.Errorf("unable to start transaction:%v", err)
+		return Prefix{}, fmt.Errorf("unable to start transaction:%w", err)
 	}
 	tx.MustExec("DELETE from prefixes WHERE cidr=$1", prefix.Cidr)
```

Comment:
Whoa, we should really replace this tx.MustExec and do proper error handling here, since MustExec panics if it fails!

Reply:
Whoa good catch, we have mustExec everywhere.

Reply:
All MustExec are remove now, this was a big mistake in the past.